cli/index.js

```diff
@@ -770,6 +771,25 @@ RainbowConfig.addOptions(
   ]
 )
 
+RainbowConfig.addOptions(
+  testingCommand
+    .command('get-bridge-on-near-balance')
+    .option(
+      '--near-receiver-account <near_receiver_account>',
+      'The account on NEAR blockchain that received the minted token.'
+    )
+    .description('Gets balance on Near for Rainbow-Bridge Eth on Near Prover.'),
+  async (args) => {
+    await getBridgeOnNearBalance(args)
+  },
+  [
+    'near-master-account',
+    'near-erc20-account',
+    'near-network-id',
+    'near-node-url',
+  ]
```

Comment:
Missing `near-receiver-account`

Offtopic of this PR, but this is a reason why we need more consistent way to handle CLI, using a framework such as yargs.